README.md

```diff
@@ -0,0 +1,74 @@
+khealth
+=====
+
+[![Docker Repository on Quay.io](https://quay.io/repository/coreos/khealth/status "Docker Repository on Quay.io")](https://quay.io/repository/coreos/khealth)
+
+**khealth** is a kubernetes cluster monitoring suite. Routines are defined which exercise various kubernetes subsystems. Collectors receive status update events from Routines and compute current state.
```

Comment:
*khealth* is a Kubernetes cluster monitoring suite. Its Routines exercise Kubernetes subsystems and send events to Collectors. Collectors collate these events to compute current cluster state. Cluster status is available over a simple HTTP API, which is served on a cluster nodeport in the example below. 

(That last sentence or something like it -- e.g., a line that says what it does, how one uses or views it.)